docs/decisions/0003-instructor-dashboard-rest-api-architecture.rst

```diff
@@ -0,0 +1,276 @@
+ADR-0003 Instructor Dashboard REST API Architecture
+===================================================
+
+Decision
+--------
+
+Implement unified Instructor Dashboard REST API with service orchestration
+pattern organizing 90 endpoints across 12 functional domains.
+
+API Endpoints & Services
+------------------------
+
+Course Information API
+^^^^^^^^^^^^^^^^^^^^^^
+
++--------+---------------------------------------------------------------+---------------------------------------------+-----------+---------------------------------------------+
+| Method | Endpoint                                                      | Services                                    | Status    | Existing API Path                           |
++========+===============================================================+=============================================+===========+=============================================+
+| GET    | /api/instructor/v1/courses/{course_key}/info                  | instructor, xmodule.modulestore             | Missing   | N/A                                         |
++--------+---------------------------------------------------------------+---------------------------------------------+-----------+---------------------------------------------+
+| PUT    | /api/instructor/v1/courses/{course_key}/info                  | instructor, xmodule.modulestore             | Missing   | N/A                                         |
```

Comment:
What capabilities will the `PUT` endpoint have?  What do we expect to be modifiable by the instructors at this endpoint?